archstrike-testing/smtp-fuzz/PKGBUILD

```diff
@@ -4,18 +4,19 @@ buildarch=1
 
 pkgname=smtp-fuzz
 pkgver=1.0
-pkgrel=3
-epoch=1
-pkgdesc="Simple smtp fuzzer"
-url="none"
-arch=(any)
+pkgrel=4
+pkgdesc='Simple smtp fuzzer'
+url='http://www.fm-berger.de/'
+arch=('any')
 depends=('perl')
-license=('GPL')
+license=('GPL2')
 groups=('archstrike' 'archstrike-fuzzers')
-source=(smtp-fuzz.pl)
+source=('http://www.fm-berger.de/download/smtpfuzz.pl')
 sha512sums=('2e63acee4e6674de31a1848da287a3784181ed01400e418c4d910d9114ad9d901abba1878f03da644341f040bd9ab19741c3437e5ccdb0701b6f39c44bf434af')
```

Comment:
is the sha512 sum of the previous source and the new one matching?

Reply:
yessir